tests/tests/CoberturaTest.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of phpunit/php-code-coverage.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\CodeCoverage\Report;
+
+use SebastianBergmann\CodeCoverage\TestCase;
+
+/**
+ * @covers \SebastianBergmann\CodeCoverage\Report\Cobertura
+ */
+class CoberturaTest extends TestCase
+{
+    public function testCloverForBankAccountTest(): void
```

Comment:
This should be `testCoberturaForBankAccountTest` instead of `testCloverForBankAccountTest`.